conf/logback.xml

```diff
@@ -44,9 +44,8 @@
     </logger>
 
     <logger name="com.google.inject" level="INFO"/>
-
     <logger name="uk.gov" level="INFO"/>
-
+    <logger name="org.apache.pekko" level="INFO"/>
```

Comment:
Why this has been added? Were you getting any errors due to the absence of this, if not then better to remove it?